### PR TITLE
first attempt for readme

### PR DESCRIPTION
**Pull request by Peeeekay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+The kurtosis extension provides enriched experience for developers writing kurtosis packages in starlark. Some of the features this extension supports are:
+1. Syntax Highlighting for Starlark
+2. Autocomplete for Kurtosis defined instructions and types
+3. Hover functionality is available
+4. Method Signature is available for kurtosis builtins 
+
+This extension is currently under continous development and may feel little rough around the edges. There are many exciting capabilites added to this extension in upcoming months; meanwhile if you encounter any issues - please report them using `kurtosis feedback` and we shall incorporate them.
+
+## Requirements
```
Comment (leeederek) [inline README.md:12]: Are there any VSCode version requirements to call out?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Kurtosis Starlark VsCode Extenstion (Official) (Pre-release)
+
+The kurtosis extension provides enriched experience for developers writing kurtosis packages in starlark. Some of the features this extension supports are:
```
Comment (leeederek) [inline README.md]:
```suggestion
The Official Kurtosis Starlark VSCode extension enriches developer experience when writing Kurtosis packages in Starlark. The specific features supported are:
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Kurtosis Starlark VsCode Extenstion (Official) (Pre-release)
```
Comment (leeederek) [inline README.md]:
```suggestion
# Kurtosis Starlark VSCode Extension (Official) (Pre-release)
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+The kurtosis extension provides enriched experience for developers writing kurtosis packages in starlark. Some of the features this extension supports are:
+1. Syntax Highlighting for Starlark
```
Comment (leeederek) [inline README.md]:
```suggestion
1. Syntax highlighting
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+2. Autocomplete for Kurtosis defined instructions and types
```
Comment (leeederek) [inline README.md]:
```suggestion
2. Auto-completion of Kurtosis defined Starlark instructions and types
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+3. Hover functionality is available
```
Comment (leeederek) [inline README.md]: @Peeeekay is this for variables, types, and instructions?

```suggestion
3. Hover functionality
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+4. Method Signature is available for kurtosis builtins
```
Comment (leeederek) [inline README.md]:
```suggestion
4. Method signature suggestions for Kurtosis defined Starlark instructions and types
```

Is this more accurate @Peeeekay ?

**Review by leeederek — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Peeeekay) [reply README.md:12]: good point; I can use this one - `1.74.2` ( that's what I currently have)

File: README.md (hunk 6 above)
Comment (Peeeekay) [reply README.md]: for functions at the moment.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# Kurtosis Starlark VSCode Extension (Official) (Pre-release)
+
+The Official Kurtosis Starlark VSCode extension enriches developer experience when writing Kurtosis packages in Starlark. The specific features supported are:
+
+1. Syntax Highlighting
+2. Auto-completion of Kurtosis defined Starlark instructions and types
+3. Hover functionality
```
Comment (leeederek) [inline README.md]:
```suggestion
3. Hover preview for functions 
```

File: README.md (hunk 1 above)
Comment (leeederek) [reply README.md:12]: That's the Nov 2022 monthly release of VSCode, I think we can just say this:
```suggestion
## Requirements
1. VSCode 1.74.2 and newer
```

**Review by leeederek — APPROVED** [review]
Good to go, after my suggestions

**Pull request merged**